icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -254,6 +254,60 @@ void dumpEnumeratedProperty(UProperty uproperty, FILE* f) {
     usrc_writeUCPTrie(f, shortPropName, utrie.getAlias(), UPRV_TARGET_SYNTAX_TOML);
 }
 
+/*
+* Export Bidi_Mirrored_Glyph values (code points) in a similar way to how enumerated
+* properties are dumped to file.
+* Note: the data will store 0 for code points without a value defined for
+* Bidi_Mirrored_Glyph.
+*/
+void dumpBidiMirroredGlyph(UProperty uproperty, FILE* f) {
+    IcuToolErrorCode status("icuexportdata: dumpBidiMirroredGlyph");
+    const char* fullPropName = u_getPropertyName(uproperty, U_LONG_PROPERTY_NAME);
+    const char* shortPropName = u_getPropertyName(uproperty, U_SHORT_PROPERTY_NAME);
+    handleError(status, fullPropName);
+
+    // Since the return value for u_charMirror(cp), when the code point doesn't
+    // have a mirroring glyph, is the code point itself, we should expect the
+    // max value to be larger than 16 bits wide.
```

Comment:
Beside the point -- since you store 0 when u_charMirror(cp) returns cp itself.

More like "Store 21-bit code points as is."